client/src/main/java/com/tigrisdata/db/client/AbstractTigrisCollection.java

```diff
@@ -100,7 +100,7 @@ protected Iterator<T> readInternal(
   protected Iterator<SearchResult<T>> searchInternal(
```

Comment:
this method is pulled upto Abstract class however it is not being used from anywhere other than StandardTigrisCollection. Can you localize it?

Reply:
done